v2/samples/insights/v1api/v1api20220615_webtest.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: insights.azure.com/v1api20220615
+kind: Webtest
+metadata:
+  name: samplewebtest
+  namespace: default
+spec:
+  tags:
+    "hidden-link:/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/aso-sample-rg/providers/microsoft.insights/components/sampleappinsights": Resource
```

Comment:
minor: Do we know that this is still required for the new webtest type?

Reply:
Good point, I'll double check and post an update.

Reply:
Double-checked. Its still needed. Getting a 400 Bad request with message below: 
`"A single 'hidden-link' tag pointing to an existing AI component is required. Found none."`